tests/foreman/cli/test_repository.py

```diff
@@ -784,7 +821,30 @@ def test_positive_synchronize_yum_repo(self):
                 self.assertEqual(new_repo['sync']['status'], 'Success')
 
     @run_only_on('sat')
-    @skip_if_bug_open('bugzilla', 1328092)
+    @tier2
```

Comment:
Think @tier1 is more appropriate, what do you think?

Reply:
It is only one component affected, so it sounds as tier1
ok, then I need to change for all other sync tests?